crawler/config_parser.py

```diff
@@ -1,9 +1,10 @@
+from __future__ import absolute_import
 import logging
 
 from configobj import ConfigObj
 from validate import Validator
 
-from utils import misc
+from .utils import misc
```

Comment:
These dots, I don't think we need them. Do we?

Reply:
Removing these might also fix the failing style tests at travis

Reply:
In Python 2 these dots are never needed but in Python 3 they might be for intrapackage imports.  https://www.python.org/dev/peps/pep-0328/

Reply:
Alright, can you please fix the travis errors then.